src/cmd/manage_nsfs.js

```diff
@@ -961,6 +962,31 @@ function validate_boolean_string_value(value) {
     return false;
 }
 
+/**
+ * validate_min_flags_for_update validates that we have at least one flag of a property to update
+ * @param {string} type
+ * @param {object} input_options_with_data
+ */
+function validate_min_flags_for_update(type, input_options_with_data) {
+    const input_options_for_update = { ...input_options_with_data };
+    // remove the config options
+    delete input_options_for_update.config_root;
+    delete input_options_for_update.config_root_backend;
+    // remove identifier
+    if (_.isUndefined(input_options_for_update.name)) {
+        // GAP - mandatory flags check should be earlier in the calls in general
+        if (type === TYPES.ACCOUNT) throw_cli_error(ManageCLIError.MissingAccountNameFlag);
+        if (type === TYPES.BUCKET) throw_cli_error(ManageCLIError.MissingBucketNameFlag);
+    }
+    delete input_options_for_update.name;
+
+    const input_options_for_update_length = Object.keys(input_options_for_update).length;
+    if (input_options_for_update_length === 0 ||
+        input_options_for_update_length === 1 && input_options_for_update.regenerate === 'false') {
+            throw_cli_error(ManageCLIError.MissingUpdateProperty);
```

Comment:
It is not the same... there 2 options where the update is not happening:
1. When passing only the `--name` flag (length is 0).
2. When passing the `--name` and `--regenerate=false` (length is 1 for a specific case).
If it would be `input_options_for_update_length <= 1 && input_options_for_update.regenerate === 'false'` then it will not fulfill the first option.

Reply:
can you add the parenthesise for the `input_options_for_update_length === 1 && input_options_for_update.regenerate === 'false'`

Reply:
Sure, will add, thanks.